net-misc/networkmanager-fortisslvpn/networkmanager-fortisslvpn-1.2.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+GNOME_ORG_MODULE="NetworkManager-${PN##*-}"
+
+inherit gnome2 autotools
+
+DESCRIPTION="NetworkManager Fortinet SSLVPN compatible plugin"
+HOMEPAGE="https://wiki.gnome.org/Projects/NetworkManager"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gtk"
+
+RDEPEND="
+	>=net-misc/networkmanager-1.2.2:=
+	>=dev-libs/dbus-glib-0.74
```

Comment:
This was copied from the networkmanager-pptp but I adapted the version number because I didn't know yet it was binary compatible. I changed it back to 1.1.

If it is not needed at all, perhaps also the -pptp ebuild should be looked at.